mm/ktsan/thr.c

```diff
@@ -221,13 +221,11 @@ void kt_thr_interrupt(kt_thr_t *thr, uptr_t pc, kt_interrupted_t *state)
 	 * BUG_ON(thr->irqs_disabled);
 	 */
 
-	memcpy(&state->stack.pc[0], &thr->stack.pc[0],
```

Comment:
This was done on a purpose: we don't copy whole stack. Now you copy whole stack. Real stack can be much smaller than capacity. Please revert.